### ISSUE TITLE
[Feature] Gutenslider plugin for Wordpress

### ISSUE THREAD

**Issue by robertsundstrom** [iolevel/wpdotnet-sdk#99]
**Description**
Gutenslider Wordpress plugin does not have a NuGet packages.

**Issues**
I have tried to add and compile the plugin source files manually. Removing unnecessary Debug classes.

The experience in the Gutenberg editor works, but the slider does not render correctly on pages.

Plugin page: https://sv.wordpress.org/plugins/gutenslider/

**Comment by robertsundstrom** [comment]
I really would like to know how to make this work.

As I said, the editor experience is working, but the slider itself does not display correctly when viewing page.

**Comment by jakubmisek** [comment]
I'm just assuming there is some generated javascript that is messed up :) do you have any javascript errors in the browsers' console?

**Comment by robertsundstrom** [comment]
@jakubmisek No, and that is strange. No, errors (or warnings). The images just display after each other on the page.

I have tried another plugin too (Getwid). The markup is rendered but the JavaScript does not run. I wonder why? 
Is it related to WP.NET?

I copied the plugin files to the directory and it compiled without problems.

**Comment by jakubmisek** [comment]
feels like something in wordpress did not enqueue the .js file to "some" array, so it's not added to the HTML code .. probably easiest would be to compare the output from regular wordpress+php with wpdotnet.

**Comment by jakubmisek** [comment]
I'm experiencing a similar issue with some themes, like it's missing images

**Comment by robertsundstrom** [comment]
Other plugins do not work either. It is in the frontend. The editor experience works.

**Comment by jakubmisek** [comment]
thanks, I still can't find the cause of it. Do you have any hints, or observations, what's going on? I would suspect something in PHP arrays, maybe reference counting going wrong.

**Comment by hez2010** [comment]
Also for syntaxhighlight involved: it works in editor but not in the published page, I have to add "brush:language" to custom css to make it display correctly.

**Comment by jakubmisek** [comment]
@hez2010 so it might be a missing CSS, seems like WordPress internally misses the stylesheet file

**Comment by robertsundstrom** [comment]
@jakubmisek I also noticed when I wrote my own plugin that the activation hook did not execute when deactivating and re-activating it. I had to restart the WP app for the hook to execute on re-activation. I assume this is not an expected behavior.
